### ISSUE TITLE
train my own data model

### ISSUE THREAD

**Issue by songyn95** [puar-playground/LRA-diffusion#1]
- If I want to train my own data model, do I need to retrain the diffusion models? How do I train if I need to?
- At the same time, I see the training scripts：train_WebVision.py, train_Food101N.py, train_Clothing1M.py, train_cifar-py， if I train my own data, which one should I choose?

I am looking forward to your reply！

**Comment by zhaihaotian** [comment]
in this paper,i think what need to train is the diffusion model to rebuild the label, and the encoder like simclr is pre-trained.
if you want to train on your own data, first of all you should have an encoder , you can use the CLIP by openai , or you can train an encoder by unsupervised learnin like SimCLR,MoCo.Then you can train the diffusion model.
